Cargo.toml

```diff
@@ -43,18 +43,19 @@ tycho-client = { git = "https://github.com/propeller-heads/tycho-indexer.git", p
 tycho-ethereum = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-ethereum", tag = "0.42.0" }
 
 # EVM dependencies
-ethabi = { version = "13.0",  optional = true }
+ethabi = { version = "13.0", optional = true }
 # some core models still rely on ethers types instead of bytes, once this is addressed this will become optional.
 ethers = { version = "2.0.13" }
-foundry-config = { git = "https://github.com/foundry-rs/foundry", rev = "57bb12e",  optional = true }
-foundry-evm = { git = "https://github.com/foundry-rs/foundry", rev = "57bb12e",  optional = true }
+foundry-config = { git = "https://github.com/foundry-rs/foundry", rev = "57bb12e", optional = true }
+foundry-evm = { git = "https://github.com/foundry-rs/foundry", rev = "57bb12e", optional = true }
 alloy-primitives = { version = "0.8.9", features = [
     "getrandom",
     "rand",
     "map-foldhash",
-], optional = true }
+] }
```

Comment:
This will eventually become optional, right?

Reply:
why?? it will be necessary for the main interfaces